.dockerignore

```diff
@@ -28,6 +28,14 @@ test/__pycache__
 __pycache__
 **/__pycache__
 static/build/**
+static/fonts/**
+static/webfonts/**
+static/ldn/**
+config_app/static/build/**
+config_app/static/fonts/**
+config_app/static/webfonts/**
+config_app/static/ldn/**
+src/**
```

Comment:
Allow building container in dir where build artifacts are present.

Reply:
Probably not related to this PR's feature but :+1:

Reply:
I'll PR this separate but let me know if issues with it or .gitignore.